src/style.css

```diff
@@ -1 +1,84 @@
-/* Styles go here */
+body {
+  margin: 0;
+}
+
+.header {
+  margin: auto 0;
+  padding: 0 50px;
+  background-color: #fff;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+
+a {
+  margin: 0;
+  padding: 0;
+}
```

Comment:
you don't need to reset margins/paddings for links, those are 0 themselves initially
```suggestion
```